test-data-server/src/services/single.service.ts

```diff
@@ -605,7 +605,7 @@ export class SingleData implements IDatabase {
                                 nickname: acc.account?.plans[i]?.nickname,
                                 servicePointIds: []
                             }
-                            if (acc.account?.openStatus == "OPEN")
+                            if (acc.account?.plans[i]?.planOverview)
                                 newPlan.planOverview = acc.account?.plans[i]?.planOverview;
                             if (acc.account?.plans[i]?.servicePointIds)
```

Comment:
This line is unnecessary

Reply:
Absolutely not. This is to prevent having a planOverview = null returned in your json.

Reply:
Well, the previous comment was about planOverview. This one is about servicePintIds, but regardless, they can be null only if the tesdata contains nulls and that was probably on purpose to test how it's being handled by the server or the client requesting this data from the endpoint, so you shouldn't interfere. Otherwise the chained nulls or undefineds are handled by the optional chaining.